c2corg_ui/static/partials/cotometer.html

```diff
@@ -0,0 +1,37 @@
+<div class="modal-body">
+  <div class="content cotometer">
+    <div class="text" translate>
+      This tool estimates the technical difficulty of a ski route (quotation toponeige). It is currently restricted to steep trails that are graded at least 4.1.
+    </div>
+    <div class="full">
+      <label translate>Skiability</label>
+      <div class="checkbox"><label><input type="radio" ng-model="cotmetCtrl.skiability" ng-value="0" ng-change="cotmetCtrl.cotometerTechnicalGrade()" /> <span translate>Wide and constant</span></label></div>
+      <div class="checkbox"><label><input type="radio" ng-model="cotmetCtrl.skiability" ng-value="0.1" ng-change="cotmetCtrl.cotometerTechnicalGrade()" /> <span translate>1 or 2 complications (steeper or narrow section...)</span></label></div>
+      <div class="checkbox"><label><input type="radio" ng-model="cotmetCtrl.skiability" ng-value="0.2" ng-change="cotmetCtrl.cotometerTechnicalGrade()" /> <span translate>Very complex (steps, narrow sections, ridges...)</span></label></div>
+    </div>
+    <div id="average-slope" class="full">
+      <label translate>Average slope (degrees)</label>
+      <input ng-model="cotmetCtrl.slope" class="form-control" placeholder="{{'Average slope in degree' | translate}}" type="number" ng-change="cotmetCtrl.cotometerTechnicalGrade()">
+    </div>
+      <div class="alert alert-danger" ng-if="cotmetCtrl.errorSlope" translate>The slope must be between 0 and 80°</div>
+    <div id="elevation" class="full">
+      <label translate>Elevation gain (meters)</label>
```

Comment:
Perhaps you could reuse some existing string, for instance ``height`` (hauteur) or ``height_diff`` (dénivelé)